deployment/constructor_params/lynx/lynx-alpha-13-child-params.json

```diff
@@ -0,0 +1,27 @@
+{
+  "LynxMockTACoApplication": {},
+  "ProxyAdmin": {},
+  "LynxTACoChildApplication": {
+    "_rootApplication": "$LynxMockTACoApplication"
+  },
+  "TransparentUpgradeableProxy": {
+    "_logic": "$LynxTACoChildApplication",
+    "admin_": "$ProxyAdmin",
+    "_data": "$EMPTY_BYTES"
+  },
+  "LynxRitualToken": {
+    "_totalSupplyOfTokens": 10000000000000000000000000
+  },
+  "Coordinator": {
+    "_application": "$TransparentUpgradeableProxy:LynxTACoChildApplication",
+    "_timeout": 3600,
+    "_maxDkgSize": 4,
+    "_admin": "0x3B42d26E19FF860bC4dEbB920DD8caA53F93c600",
```

Comment:
What about `$DEPLOYER` keyword?

Reply:
Hehe this was something we played around with but didn't include in the PR - <https://github.com/derekpierre/nucypher-contracts/commit/b710fab52c2213f35834f775f90dc007b98ff4fb>

We felt nervous doing implicit deployer in the params file vs explicit.

We can potentially add it if you like. The above code is based on old code, but can be reworked.

Reply:
Great minds think alike... @derekpierre actually has a commit for this.  We'll include it!